packages/conform-yup/index.ts

```diff
@@ -63,15 +63,15 @@ export function getFieldsetConstraint<Source extends yup.AnyObjectSchema>(
 								case 'min':
 									if (
 										!constraint.min ||
-										constraint.min < Number(test.params?.min)
+										Number(constraint.min) < Number(test.params?.min)
```

Comment:
I casted these, because the original type said they could be a number or string, let me know if you were expecting something else and I'll update accordingly.